common/base/kustomization.yaml

```diff
@@ -3,3 +3,4 @@ kind: Kustomization
 resources:
   - ../helm-repositories/
   - ../values/
+  - ../kommander-operator/
```

Comment:
this should be done for airgapped envs

Reply:
But it is. `base` is applied always here:

https://github.com/mesosphere/kommander-cli/blob/main/pkg/installer/git-repository/clusters/kommander_host-cluster/common/kustomization.yaml

Reply:
there is check here though  [ this check here ](https://github.com/mesosphere/kommander-cli/blob/c632dca506262d2e96bef879c4c3c53dc54da106/pkg/installer/helmrepositories.go#L45)

Reply:
I guess you're right! this installation command is so confusing :smile:

Reply:
> there is this check here though [ this check here ](https://github.com/mesosphere/kommander-cli/blob/c632dca506262d2e96bef879c4c3c53dc54da106/pkg/installer/helmrepositories.go#L45) thinking

@Azhovan this `airgapped` is just for helmrepositories to work:
https://github.com/mesosphere/kommander-applications/blob/main/common/airgapped/kustomization.yaml